rex_xai/input_data.py

```diff
@@ -198,7 +210,12 @@ def set_mask_value(self, m):
             case "mean":
                 self.mask_value = tt.mean(self.data).item()  # type: ignore
             case "spectral":
-                self.mask_value = lambda m, d: spectral_occlusion(m, d, device=self.device)
+                self.mask_value = lambda m, d: spectral_occlusion(
+                    m, d, device=self.device
+                )
+            case "context":
+                self.mask_value = lambda m, d: context_occlusion(m, d, self.context)
+                # TODO: Add args for noise and setting the context as currently only available through custom script
```

Comment:
Spotted a TODO - if this is outside the scope of this PR and intentionally left for later I'd suggest opening an issue to track it!